interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +841,22 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+<?php require_once($GLOBALS['srcdir']."/validation/validation_script.js.php");?>
+<?php $collectThis=collectValidationPageRules("add_edit_issue",$_SERVER['PHP_SELF'])?>
```

Comment:
That sounds reasonable. Just make sure the # character in the option_id doesn't throw errors or break things when do save in the Administration->Lists->Page Validation.
special characters are generally frowned upon in the option_id, but in this case would be well worth it. Since you now will basically have a modular piece of code that will be the same to handle your validation. I also recommend using the active toggle(mysql column) in list_options when doing the lookup, thus can turn off validation sets on pages if the user wishes without needing to delete them. This also allows you to add a couple examples which are not turned on by default to give users an idea of how to use it.